docs/src/content/docs/ru/guides/customization.mdx

```diff
@@ -195,7 +195,7 @@ defineConfig({
 Starlight имеет встроенную поддержку для добавления ссылок на ваши аккаунты в социальных сетях в заголовок сайта
 через параметр [`social`](/ru/reference/configuration/#social) в интеграции Starlight.
 
-В настоящее время поддерживаются ссылки на Bitbucket, Codeberg, CodePen, Discord, Email, Facebook, GitHub, GitLab, Gitter, Instagram, LinkedIn, Mastodon, Microsoft Teams, RSS-ленту, Stack Overflow, Telegram, Threads, Twitch, Twitter, X и Youtube.
+В настоящее время поддерживаются ссылки на Bitbucket, Codeberg, CodePen, Discord, Email, Facebook, GitHub, GitLab, Gitter, Instagram, LinkedIn, Mastodon, Microsoft Teams, Patreon, Reddit, RSS-ленту, Stack Overflow, Telegram, Threads, Twitch, Twitter, X и Youtube.
```

Comment:
I think this entire sentence could be removed by updating the translation with the changes of #869.

Reply:
👌